media-gfx/gscan2pdf/gscan2pdf-1.7.1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DIST_TEST="do"
+
+inherit eutils perl-module
+
+DESCRIPTION="Scan documents, perform OCR, produce PDFs and DjVus"
+HOMEPAGE="http://gscan2pdf.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="test"
+
+DEPEND="test? (
+		app-text/pdftk
+		app-text/poppler
+		dev-perl/Sub-Override
+		dev-perl/Test-Pod
+		media-libs/fontconfig
+		x11-libs/gdk-pixbuf[tiff]
+
+		app-text/djvu[tiff]
+		app-text/tesseract[-opencl,osd,tiff]
+		app-text/unpaper
+		media-gfx/imagemagick[-opencl,djvu,png,tiff,perl]
+		media-gfx/sane-backends[sane_backends_test]
+		media-gfx/sane-frontends
+	)"
+# Tests fail on sandbox violation with OpenCL ImageMagick and Tesseract
```

Comment:
The sandbox violation is not a gscan2pdf problem, just the normal OpenCL one that the tests need access to the GPU device, in this case reported as: `X server found. dri2 connection failed! `

There seem to be a number of different options for fixing, plus the problem that different GPUs use different device paths. e.g.
https://bugs.gentoo.org/show_bug.cgi?id=500580#c5
https://bugs.gentoo.org/show_bug.cgi?id=604168

I can create and test a fix for Intel hardware. But some pointers for the best way to go about it would be good - we probably need an OpenCL guy too!

Indentation fix will be in the next version.